### PR TITLE
feat: tutorial for btcd simnet

### PR DESCRIPTION
**Pull request by vuvoth**
## Description
This is basic command line to start btc simnet.

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
    
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary

### CODE REVIEW

**Review by sczembor — COMMENTED** [review]
I suggest to move it to `docs` directory as a separate file. we can link it here to there

**Review by robert-zaremba — COMMENTED** [review]
you can run `markdownlint-cli2 --fix <filename>` to automatically fix the linter issues

**Comment by vuvoth** [comment]
> I suggest to move it to `docs` directory as a separate file. we can link it here to there

I think we should keep it here and link from the doc to this folder. To make the development flow easier, we will add a Docker or script to this folder. So, this makes more sense if the document and script or Docker file are in the same folder.

File: .gitignore [hunk 1]
```diff
@@ -23,3 +23,6 @@ go.work.sum
 
 # binaries
 out
+
+contrib/bitcoin
+contrib/regtest
```
Comment (robert-zaremba) [inline .gitignore:28]: Let's change to simtest?

File: contrib/README.md
Comment (robert-zaremba) [inline contrib/README.md:1]: Let's rename this file to `Bitcoin_mock.md`

File: contrib/README.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+# Run btcd simnet mode
+
+## Install btcd and btcwallet
+
+Follow the instruction in [btcd README.md](https://github.com/btcsuite/btcd?tab=readme-ov-file#installation)
+and [btcwallet README](https://github.com/btcsuite/btcwallet?tab=readme-ov-file#installation-and-updating) for install btcd and btcwallet
```
Comment (robert-zaremba) [inline contrib/README.md:6]:
```suggestion
and [btcwallet README](https://github.com/btcsuite/btcwallet?tab=readme-ov-file#installation-and-updating) to install btcd and btcwallet
```

File: contrib/README.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+## Run btcd with simnet mode
+
+### Bootstrap the wallet and btc node
+
+First you need start the btcd in simnet mode
```
Comment (robert-zaremba) [inline contrib/README.md:12]:
```suggestion
First you need to start the btcd in simnet mode
```

File: contrib/README.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+```bash
+btcd --simnet --rpcuser=user --rpcpass=password
+```
+
+Create btc wallet. Please remember password we can use this somewhere.
```
Comment (robert-zaremba) [inline contrib/README.md:18]:
```suggestion
Create btc wallet. Please remember the password.
```

File: contrib/README.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+```bash
+btcwallet --simnet --create
+```
+
+Connect you wallet to btcd simmode. The password and user here must be the btcd password/user pair.
+Open a new terminal and run command below:
```
Comment (robert-zaremba) [inline contrib/README.md:25]:
```suggestion
Open a new terminal and run:
```

File: contrib/README.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+```bash
+btcwallet --simnet -u=user -P=password
+```
+
+Create a new address we will use this for `--miningaddr`, the address receive btc when mining a new block.
```
Comment (robert-zaremba) [inline contrib/README.md:31]:
```suggestion
Create a new address. We will use it as the `--miningaddr` parameter - an address that receives btc when mining a new block.
```

File: contrib/README.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+Open a new terminal and run command below:
```
Comment (robert-zaremba) [inline contrib/README.md:32]:
```suggestion
Open a new terminal and run:
```

File: contrib/README.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+
+```bash
+btcctl --simnet --wallet --rpcuser=user --rpcpass=password getnewaddress
+```
+
+Copy the address created by command above. Shutdown the btcd service and run with command below:
```
Comment (robert-zaremba) [inline contrib/README.md:38]:
```suggestion
Copy the address created by the command above. Shutdown the btcd service and run:
```

**Review by robert-zaremba — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (vuvoth) [reply .gitignore:28]: Actually, we store data in ~/.btcd. :D This .gitignore for bitcoind I wanted to create before.

**Pull request merged**